modules/boot/include/settings.h

```diff
@@ -10,6 +10,7 @@
 #define LOAD_FILE 1
 
 extern bool g_dropShadows;
+extern bool g_angle;
 extern uint32_t g_fontType;
```

Comment:
can try to be more descriptive here. just "angle" could mean alot of things
since it seems like value 1 (true) is decimal, it can be `g_angleValuesInDecimal`